test/value_init_test3.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Peter Dimov
+// Distributed under the Boost Software License, Version 1.0.
+// https://www.boost.org/LICENSE_1_0.txt)
+
+#include <boost/utility/value_init.hpp>
+#include <boost/core/lightweight_test.hpp>
+#include <boost/config.hpp>
+#include <boost/config/pragma_message.hpp>
+
+#if __cplusplus >= 201103L || ( defined(BOOST_MSVC) && BOOST_MSVC >= 1900 )
```

Comment:
Doesn't `BOOST_NO_CXX14_AGGREGATE_NSDMI` cover this?

Reply:
No.

Reply:
BOOST_NO_CXX14_AGGREGATE_NSDMI is about the C++14 feature, the equivalent of __cpp_aggregate_nsdmi. What I need here is the nonexistent macro about the C++11 feature, the equivalent of __cpp_nsdmi. No idea why we have the former in Config, but not the latter.

Reply:
Ok, thanks. Maybe we should add one.